Formula/php.rb

```diff
@@ -454,3 +438,37 @@ def plist; <<~EOS
  ])
 
  dnl
+@@ -487,7 +491,11 @@ dnl add an include path.
+ dnl if before is 1, add in the beginning of INCLUDES.
+ dnl
+ AC_DEFUN([PHP_ADD_INCLUDE],[
+-  if test "$1" != "/usr/include"; then
++  case "$1" in
++  "/usr/include"[)] ;;
++  /Library/Developer/CommandLineTools/SDKs/*/usr/include[)] ;;
++  /Applications/Xcode*.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*/usr/include[)] ;;
++  *[)]
+     PHP_EXPAND_PATH($1, ai_p)
+     PHP_RUN_ONCE(INCLUDEPATH, $ai_p, [
+       if test "$2"; then
+@@ -495,8 +503,8 @@ AC_DEFUN([PHP_ADD_INCLUDE],[
+       else
+         INCLUDES="$INCLUDES -I$ai_p"
+       fi
+-    ])
+-  fi
++    ]) ;;
++  esac
+ ])
+
+ dnl internal, don't use
+@@ -2411,7 +2419,8 @@ AC_DEFUN([PHP_SETUP_ICONV], [
+     fi
+
+     if test -f $ICONV_DIR/$PHP_LIBDIR/lib$iconv_lib_name.a ||
+-       test -f $ICONV_DIR/$PHP_LIBDIR/lib$iconv_lib_name.$SHLIB_SUFFIX_NAME
++       test -f $ICONV_DIR/$PHP_LIBDIR/lib$iconv_lib_name.$SHLIB_SUFFIX_NAME ||
```

Comment:
I'm wondering if we should make this an inreplace line instead of a patch, might be easier to manage over time.

Reply:
You could perhaps handle it with inreplace, but we still have to support both `.dylib` and `.tbd` suffixes exposed by supported macOS platforms.

Reply:
But `/usr/include/iconv.h` exists in Mojave at least since they restored `/usr/include` in 10.14.1 .

Reply:
If they truly did "about face" on that one it would be fine, but my upgraded mojave machine does not have a `/usr/include`. I still think it requires an additional package to be installed to get them. I have XCode, but not CLT on 10.14.1.

Reply:
And I have CLT but not XCode so it might be clt related then since I've not installed anything in addition to make this work.

Reply:
Apple distributed CLT of High Sierra by software update for Mojave.
It removes `/Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg` and restores `/usr/include` .

Reply:
I just found out that supposedly there might be a bug that did just that so it doesn't seem to happen on all systems. Do you have any links to any documented information about it @mizunashi-mana ?

Reply:
They seem to have taken the affected package out of distribution now so it shouldn't happen anymore.

Reply:
@javian Actually, I'm waiting official announce and cannot find any documents.  However, the summary said such when I updated.  We can see it by System Information > Software > Install.  The latest CLT update shows `Command Line Tools (macOS High Sierra version 10.13) for Xcode` with version 10.1 .